styles/mixins/button.js

```diff
@@ -1,11 +1,15 @@
 module.exports = function () {
   return function (mixin, bgColor, color) {
-    return {
-      'background-color': bgColor,
-      color,
-      display: 'inline-block',
-      'line-height': 1,
-      'text-align': 'center'
+    const activeBgColor = 'red';
```

Comment:
It's just a placeholder for testing. I mentioned this in the room but not sure anyone saw it. I'd like to add https://www.npmjs.com/package/postcss-colour-functions. We can use it for the hover effects on buttons. So `activeBgColor` would be `activeBgColor = darken(bgColor)`